.github/workflows/provenance.yml

```
@@ -21,6 +21,6 @@ jobs:
          cache: npm
      - run: npm install -g npm@latest
      - run: npm ci
-     - run: npm publish --provenance --access public
+     - run: npm publish
```

Think we still need `--access public`:

```suggestion
     - run: npm publish --access public
```